plugins/factory-plugin/src/workspace-projects-manager.ts

```diff
@@ -57,4 +63,63 @@ export class WorkspaceProjectsManager {
         theia.window.showInformationMessage("Che Workspace: Finished cloning projects.");
     }
 
+    async syncWorkspace() {
```

Comment:
`async addSyncWatchers` or `async addFileSystemWatchers`?

Reply:
`startSyncWorkspaceProjects` ? I don't like to mentionned the how it is implemented but trying to describe what it is doing.

Reply:
`startSyncWorkspaceProjects` is OK